src/scripts/util.sh

```diff
@@ -91,3 +91,22 @@ is_renewal_required() {
     is_finshed_week_sec=$(( ($one_week_sec - $last_renewal_delta_sec) ))
     [ $is_finshed_week_sec -lt 0 ]
 }
+
+# symlinks any *.conf files in /etc/nginx/user.conf.d
+# to /etc/nginx/conf.d so they are included as configs
+# this allows a user to easily mount their own configs
+link_user_configs() {
+    SOURCE_DIR=${1-/etc/nginx/user.conf.d}
```

Comment:
Surround in quotes to allow for directories with spaces in them (yes, people that do that bring pain upon themselves, but let's be good neighbors)